src/sources/journald.rs

```diff
@@ -297,6 +304,7 @@ impl Default for JournaldConfig {
             journalctl_path: None,
             journal_directory: None,
             journal_namespace: None,
+            extra_args: vec![],
```

Comment:
nit: For future work, `vec![]` is just `Vec::new()`, which avoids using a macro.